src/plugins/intel_cpu/src/emitters/utils.hpp

```diff
@@ -0,0 +1,87 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <cpu/x64/jit_generator.hpp>
+
+namespace ov {
+namespace intel_cpu {
+
+// Usage
+// 1. Include this headfile where JIT kennels of CPU plugin are implemented for Register printing
+// 2. Invoke RegPrinter::print method. Here are some examples. Note that if register name is not set,
+//    original Xbyak register name will be printed by default.
+//    Example 1:
+//    Invocation: RegPrinter::print<float>(*this, vmm_val, "vmm_val");
+//    Console:    vmm_val: {30, 20, 25, 29, 24, 31, 27, 23}
```

Comment:
Thinking about console output: it sounds like it would always good to print also register type and number (like ymm0) even if name provided. Like:
ex if name provided: `"vmm_val" | <ymm0>: {...}`
ex if name not provided: `<ymm0>: {...}`

Reply:
Applied. Thanks Dmitry!